### ISSUE TITLE
Extract to procedure: Rename not possible if variables are moved

### ISSUE THREAD

**Issue by NKarolak** [DavidFeldhoff/al-codeactions#95]
If you extract lines of code without a variable to be moved to a procedure, you can rename the new procedure directly:
![E2P_1](https://user-images.githubusercontent.com/34504100/98938942-f668ad80-24e8-11eb-81cc-9aadbc5a37a5.gif)

However, if you include a variable that will be moved, you cannot rename the procedure anymore within the same step. As you can see, `newProcedure` is missing within the entry page:
![E2P_2](https://user-images.githubusercontent.com/34504100/98939146-447db100-24e9-11eb-80fe-4a86e40b9bc4.gif)
So, whenever a variable would be moved, you actively need to go to the newProcedure call and rename it with F2.

Code:
```
local procedure TestFunction();
    var
        i: integer;
    begin
        if true then begin
            // i := 1;
        end;
    end;
```

**Comment by DavidFeldhoff** [comment]
Hi Natalie, thanks for the hint! I'll fix it tomorrow. 
Related to #92

**Comment by DavidFeldhoff** [comment]
Hi Natalie, I just fixed this one in release 0.2.21.
Thanks for not getting tired of reporting :)

**Comment by NKarolak** [comment]
Thank you for not getting tired of fixing and still staying friendly ;-)
